src/react-apps/applications/service-development/src/App.tsx

```diff
@@ -2,26 +2,34 @@
 // https://github.com/facebook/create-react-app/issues/4801#issuecomment-409553780
 // Disabled for React Router rendering
 
+/* tslint:disable:jsx-boolean-value */
+// Extensive used in Material-UI's Grid
+
 import Grid from '@material-ui/core/Grid';
+import Hidden from '@material-ui/core/Hidden';
+import { createMuiTheme, createStyles, MuiThemeProvider, withStyles, WithStyles } from '@material-ui/core/styles';
 import * as React from 'react';
 import { connect } from 'react-redux';
+import { HashRouter as Router, Redirect, Route } from 'react-router-dom';
 import LeftDrawerMenu from '../../shared/src/navigation/drawer/LeftDrawerMenu';
-import AppBarComponent from '../../shared/src/navigation/main-header/app-bar';
-import SubApp from '../../ux-editor/src/SubApp';
+import AppBarComponent from '../../shared/src/navigation/main-header/appBar';
+import altinnTheme from '../../shared/src/theme/altinnStudioTheme';
 import NavigationActionDispatcher from './actions/navigationActions/navigationActionDispatcher';
 import './App.css';
+import { redirects } from './config/redirects';
+import { routes } from './config/routes';
 
-import { HashRouter as Router, Redirect, Route } from 'react-router-dom';
+export interface IAppProps extends WithStyles<typeof styles> { }
```

Comment:
`typeof`? Isn't styles an Object?

And this interface doesn't extend anything. You can just use the `WithStyles<type>` in the class decoration.

Reply:
https://material-ui.com/guides/typescript/#augmenting-your-props-using-withstyles

Reply:
Still doesn't extend with anything, so no need to have it.